lib/template-mixins.js

```diff
@@ -414,6 +422,8 @@ module.exports = function (fields, options) {
         _.each(mixins, function (mixin, name) {
             var handler = _.isFunction(mixin.handler) ? mixin.handler : function () {
                 return function (key) {
+                    this.options = this.options || {};
+                    this.options.fields = this.options.fields || {};
```

Comment:
Is it worth passing in field as well rather than setting in the renderWith functions?